scripts/src/client.ts

```diff
@@ -129,19 +132,23 @@ export class ClientRequests {
 
 	private getConfig() {
 		return {
-			headers: {
+			headers: Object.assign({
 				"x-request-id": uuid(),
 				"Authorization": this.auth ? `Bearer ${ this.auth.token }` : "",
-			},
+			}, this.headers),
 		};
 	}
 }
 
+type ClientConfig = { headers?: StringMap, sdkVersion?: string };
+
 export class Client {
-	public static async create(signInPayload: SignInPayload, config?: { headers?: StringMap }): Promise<Client> {
+	public static async create(signInPayload: SignInPayload, config: ClientConfig = {}): Promise<Client> {
 		if (!this.blockchainConfig) {
 			this.blockchainConfig = (await ClientRequests.getConfig()).blockchain;
 		}
+		config.headers = config.headers || {};
```

Comment:
you don't need this line if the config defaults {}

Reply:
But `headers` might be missing. It's a ramanant that's left after some refactoring, I can change the default to `{headers:{}}` now.